test_common.c

```diff
@@ -51,6 +51,18 @@ int main(int argc, char **argv)
     } else
         printf("REF mechanism\n");
 
+    char *Top = word;
+    char *pool = NULL;
+
+    if (CPYmask) {
```

Comment:
You can write comment(s) for `CPYmask`. That is, the memory pool is only allocated for REF mechanism.